browser/speedreader/speedreader_browsertest.cc

```diff
@@ -62,9 +59,6 @@ class SpeedReaderBrowserTest : public InProcessBrowserTest {
   void SetUpCommandLine(base::CommandLine* command_line) override {
     base::FilePath test_data_dir;
```

Comment:
not used

Reply:
fixed in https://github.com/brave/brave-core/pull/10090/commits/a0a67b46ad0bae8267e92fc8868de410a7851b81